initializr-web/src/test/java/io/spring/initializr/web/autoconfigure/InitializrAutoConfigurationTests.java

```diff
@@ -57,6 +169,45 @@ public void customRestTemplateBuilderIsUsed() {
 				});
 	}
 
+	@Test
+	public void webConfiguration() {
+		WebApplicationContextRunner contextRunner = new WebApplicationContextRunner()
```

Comment:
I think I'd call that `webContextRunner` or something. I am not sure I like the idea of using the same name as the general purpose field.

Reply:
👍